rust/wal3/src/bin/wal3-scrub.rs

```diff
@@ -0,0 +1,60 @@
+use arrrg::CommandLine;
+use biometrics::Sensor;
+//use object_store::aws::{AmazonS3, AmazonS3Builder, S3ConditionalPut};
+
+use wal3::{LogReader, LogReaderOptions};
+
+#[derive(Clone, Eq, PartialEq, arrrg_derive::CommandLine)]
+pub struct Options {
+    #[arrrg(optional, "Path to the object store.")]
+    pub path: String,
+    #[arrrg(nested)]
+    pub log: LogReaderOptions,
+}
+
+impl Default for Options {
+    fn default() -> Self {
+        Options {
+            path: "wal3.data".to_string(),
+            log: LogReaderOptions::default(),
+        }
+    }
+}
+
+#[tokio::main(flavor = "multi_thread", worker_threads = 2)]
+async fn main() {
+    let (options, free) = Options::from_command_line_relaxed("USAGE: wal3 [OPTIONS]");
```

Comment:
nit: I'd prefer this to be under the `examples` folder. From my understanding this file does not contain the logic for scrubbing, and it only showcases how the scrubbing functionality can be used.

Reply:
This is a tool that we will want to run.  Why would it be an example?

Reply:
From my understanding a scrubbing tool should take in parameters from the command line and scrub the WAL3 instance according to the config. Optionally there could be more features of the tool such as periodic scrubbing. The current impl only scrubs a local object store once, which looks more like an usage example instead of a feature complete tool.

Reply:
update: it's fine to have it here for now